src/main/java/MinesweeperController.java

```diff
@@ -99,19 +124,20 @@ private void placeMines() {
     public boolean isOnGrid (int x, int y){
         int minVal = 0;
         int maxVal = boardSize - 1;
-        if ( x < minVal || x > maxVal || y < minVal ||  y > maxVal){
-            return false;
-        }
-        return true;
+        return x >= minVal && x <= maxVal && y >= minVal && y <= maxVal;
     }
     public void playMove(MinesweeperCell clickedCell) {
         int cellValue = clickedCell.value;
-        if (cellValue == -1) { // mine
+        if (cellValue == -1) {
+            // mine
+            clickedCell.setGraphic(bomb);
             //reveal all cells
```

Comment:
still need to reveal all cells, probably create a function that 'uncovers' each cell, showing either a bomb, empty , or a number